core/src/main/java/com/graphhopper/routing/profiles/Toll.java

```diff
@@ -31,6 +33,16 @@
         this.name = name;
     }
 
+    public static Toll find(String name) {
```

Comment:
This seems to be unused

Reply:
As the others have this, we could keep it? But we can also remove if you prefer it.

Reply:
Ok no then it makes sense to keep it